src/pages/login/index.tsx

```diff
@@ -0,0 +1,15 @@
+import LoginButton from '../../components/atoms/LoginButton';
```

Comment:
```suggestion
import { LoginButton } from '@atoms';
```
path alias 설정을 해둬서 이렇게 사용하시면 될 것 같습니다!

또한 atom들을 한번에 import 할 수 있도록 atom 안에 index.ts 라는 파일에서 한번에 export를 해주고 있습니다!
```javascript
// atoms/index.ts

import Button from './Button';
import LoginButton from "./LoginButton";

export { Button, LoginButton };
```
와 같이 추가해주시면 위와 같이 import할 수 있습니다!

글로 설명하다보니 좀 복잡해 보이는 것 같아 이해안되시면 말씀해주세요! 디코로 달려가겠습니다.

참고로 Path alias 설정은 다음과 같고 tsconfig.json파일에서 확인하실 수 있습니다!

```javascript
// tsconfig.json

    "baseUrl": "./",
    "paths": {
      "@assets/*": ["src/assets/*"],
      "@pages/*": ["src/pages/*"],
      "@atoms": ["src/components/atoms"],
      "@molecules": ["src/components/molecules"],
      "@organisms": ["src/components/organisms"],
      "@styles/*": ["src/styles/*"],
      "@typings/*": ["src/typings/*"],
      "@utils/*": ["src/utils/*"],
      "@apis/*": ["src/apis/*"]
    }
```

Reply:
아하 paths 설정을 다 주셨군요!! index에서 한번에 export하다니 꽤 편리하네요!! 태승님 덕분에 진짜 많은 거 알아가는 것 같아 정말 다행인 것 같습니다! 함께 토이플젝을 진행해주셔서 진짜 감사합니다 ㅎㅎ